Sources/MapboxCoreNavigation/AlternativeRouteDetectionOptions.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+/**
+ Options to configure fetching and detecting `AlternativeRoute`s during navigation.
+ */
+public enum AlternativeRouteDetectionOptions {
+    /**
+     Describes conditions when requests for new `AlternativeRoute`s will be made.
+     */
+    public struct DetectionStrategy {
+        /**
+         Enables requesting for new alternative routes after passing a fork (intersection) where another alternative route branches.
+
+         Default value is `true`.
+         */
+        public var refreshAfterPassingDeviation = true
```

Comment:
Property names should describe rather than command. These properties’ names should begin with `refreshes`.